packages/lambda-api-tracker/src/index.ts

```diff
@@ -1,20 +1,33 @@
+import { Projection } from '@basemaps/geo';
 import {
+    Const,
+    HttpHeader,
     LambdaHttpResponse,
     LambdaHttpResponseCloudFront,
+    LambdaHttpResponseCloudFrontRequest,
     LambdaSession,
     LogType,
-    Router,
+    tileFromPath,
+    populateAction,
+    TileType,
 } from '@basemaps/lambda-shared';
 import { CloudFrontRequestEvent } from 'aws-lambda';
+import { queryStringExtractor } from './query';
 import { ReqInfoCloudFront } from './req-info-cloudfront';
-import tile from './tile-request';
+import { ValidateRequest } from './validate';
 
-const app = new Router(
-    (status: number, description: string, headers?: Record<string, string>): LambdaHttpResponseCloudFront =>
-        new LambdaHttpResponseCloudFront(status, description, headers),
-);
+const projection = new Projection(256);
 
-app.get('tiles', tile);
+const setTileSession = (info: ReqInfoCloudFront): void => {
+    const xyzData = tileFromPath(info.rest);
+    if (xyzData != null && xyzData.type === TileType.Image) {
```

Comment:
could simplify to `if(xyzData?.type == ...)`

Reply:
I'll get used to `?.` soon I promise